projects/plugins/jetpack/_inc/blogging-prompts.php

```diff
@@ -51,6 +51,15 @@ function jetpack_setup_blogging_prompt_response( $post_id ) {
  * Utility functions.
  */
 
+/**
+ * Get the 'jetpack_blogging_prompts_enabled' and default to true if the site appears to be a blog.
+ *
+ * @return boolean
+ */
+function jetpack_are_blogging_prompts_enabled() {
```

Comment:
nice, so with this change we can now also manually enable prompts for non-"blog" sites 🎉